sklearn/metrics/pairwise.py

```diff
@@ -1015,19 +1015,34 @@ def paired_euclidean_distances(X, Y):
 
 
 def paired_manhattan_distances(X, Y):
-    """Compute the L1 distances between the vectors in X and Y.
+    """Compute the L1 distances between X and Y.
+
+    Distances are calculated between (X[0], Y[0]), (X[1], Y[1]), ... .
```

Comment:
To be a little more consistent with `paired_distances`:

```suggestion
    Distances are calculated between (X[0], Y[0]), (X[1], Y[1]), etc...
```

Reply:
Actually not a fan of the notation in `paired_distances` in this case -- I find three dots that indicate the continuation of a series to be more intuitive than `etc ...`. The former notation is what I'm used to from maths.

How do you think about the notation `(X[0], Y[0]), (X[1], Y[1]), ..., (X[n_samples], Y[n_samples]).`? Might be even more descriptive?

Reply:
Yes, I do like the `..., (X[n_samples], Y[n_samples])` more. It will likely require a new line to stay within the 88 character limit:

```suggestion
    Distances are calculated between (X[0], Y[0]), (X[1], Y[1]), ...,
    (X[n_samples], Y[n_samples]).
```

Reply:
Great catch, thank you for your review @thomasjpfan !